test/intl402/DurationFormat/prototype/format/branding.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.format
+description: Verifies the branding check for the "format" function of the DurationFormat prototype object.
+features: [Intl.DurationFormat]
+---*/
+
+const format = Intl.DurationFormat.prototype.format;
+
+assert.sameValue(typeof format, "function");
+
+assert.throws(TypeError, () => format.call(undefined), "undefined");
```

Comment:
I learned from https://github.com/tc39/test262/issues/3224 that this kind of test should call the method with otherwise correct arguments.

e.g.
```suggestion
assert.throws(TypeError, () => format.call(undefined, duration), "undefined");
```
(with some `duration` defined above)